spec/features/staff/beis_users_can_edit_a_submission_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
+
+RSpec.feature "BEIS users can edit a submission" do
+  let(:delivery_partner_user) { create(:delivery_partner_user) }
+  let!(:submission) { create(:submission, organisation: delivery_partner_user.organisation, deadline: nil, description: "Legacy Submission") }
+
+  context "Logged in as a BEIS user" do
+    scenario "they can edit a Submission to set the deadline" do
+      PublicActivity.with_tracking do
+        user = create(:beis_user)
+        authenticate!(user: user)
+
+        visit organisation_path(user.organisation)
+
+        within "##{submission.id}" do
+          click_on I18n.t("default.link.edit")
+        end
+
+        fill_in "submission[deadline(3i)]", with: "31"
+        fill_in "submission[deadline(2i)]", with: "1"
+        fill_in "submission[deadline(1i)]", with: "2021"
+
+        click_on I18n.t("default.button.submit")
+
+        expect(page).to have_content I18n.t("action.submission.update.success")
+        within "##{submission.id}" do
+          expect(page).to have_content("31 Jan 2021")
+        end
+        auditable_events = PublicActivity::Activity.where(trackable_id: submission.id)
```

Comment:
I think we often have a seperate spec for the public activity to document the behaviour, like this one:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/30a244379a6640fc57f9ee3ab7b15401fd4d648e/spec/features/staff/users_can_create_a_budget_spec.rb#L21

I like the explicitness of this approach, what do you think?